src/styles/blocks/page.scss

```diff
@@ -0,0 +1,31 @@
+.page {
```

Comment:
make sure to add the

`.page:has(.page__menu:target) {
  overflow: hidden; /* not to scroll the page */
}`

so the user can't scroll the page while the menu is open.